examples/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dev-dependencies]
 relm4 = { path = "../relm4", features = ["macros"] }
+relm4-components = { path = "../relm4-components" }
```

Comment:
I think we should rather place it here, right? https://github.com/Relm4/Relm4/tree/main/examples/relm4-components

Reply:
Yes, fixed.